src/liquid/components/ld-sidenav/ld-sidenav-back/ld-sidenav-back.css

```diff
@@ -0,0 +1,135 @@
+.ld-sidenav-back {
+  /* layout */
+  --ld-sidenav-back-border-radius: var(--ld-br-l);
+  --ld-sidenav-back-icon-size: var(--ld-sp-40);
+
+  /* colors */
+  --ld-sidenav-back-col: var(--ld-col-neutral-800);
+  --ld-sidenav-back-indicator-col: transparent;
+
+  outline: none;
+  display: block;
+
+  @media (hover: hover) {
+    &:where(:hover) {
+      --ld-sidenav-back-col: var(--ld-thm-primary);
+      --ld-sidenav-back-indicator-col: var(--ld-col-neutral-300);
+    }
+  }
+
+  &:focus:focus-visible {
+    --ld-sidenav-back-col: var(--ld-thm-primary);
+    --ld-sidenav-back-indicator-col: var(--ld-thm-primary-focus);
+  }
+
+  &:active {
+    --ld-sidenav-back-col: var(--ld-thm-primary-active);
+  }
+
+  &--rounded {
+    --ld-sidenav-back-border-radius: calc(
+      0.5 * var(--ld-sidenav-back-icon-size)
+    );
+  }
+
+  ::slotted(ld-sidenav-navitem) {
+    margin-top: var(--ld-sidenav-padding-y);
+    margin-bottom: var(--ld-sidenav-padding-y);
+  }
+}
+
+.ld-sidenav-back__btn-back {
+  display: none;
+  width: calc(100% - 2 * var(--ld-sidenav-padding-x));
+  background-color: transparent;
+  position: relative;
+  font: var(--ld-typo-body-s);
+  border: 0;
+  border-radius: var(--ld-sidenav-back-border-radius);
+  cursor: pointer;
+  user-select: none;
+  touch-action: manipulation;
+  color: var(--ld-sidenav-back, var(--ld-col-neutral-800));
+  grid-template-columns: var(--ld-sidenav-back-icon-size) 1fr;
+  gap: var(--ld-sp-12);
+  align-items: center;
+  font-weight: 700;
+  line-height: 1;
+  box-sizing: border-box;
+  padding: 0;
+  text-align: left;
+  outline: none;
+  margin: var(--ld-sidenav-padding-y) var(--ld-sidenav-padding-x);
```

Comment:
This is style applied to an element within the shadow DOM of `ld-sidenav-back`. It doesn't make sense to move these styles to the `ld-sidenav` component. Keeping as is.

Reply:
You only have to separately style the back button, because you do not move the margins away from the navitem and let the `ld-sidenav` component handle these spacings. I know it's details, but I think it'd make things a little easier. Maybe I can show you what I mean in a quick 1-on-1 session?

Reply:
I still see this as a relevant issue, especially as it's a little confusing that the background-color area doesn't match with the clickable area. But this can be done some time later in a refactoring, if desired. Resolving this issue conversation.